liquibase-dist/src/main/archive/liquibase

```diff
@@ -54,10 +56,12 @@ else
   done
 
   # Gather jars from a local liquibase_libs directory
-  CP="$CP""$CP_SEPARATOR""liquibase_libs/"
-  for i in liquibase_libs/*.jar; do
-    CP="$CP""$CP_SEPARATOR""$i"
-  done
+  if [[ -f ./liquibase_libs/*.jar ]]
+    CP="$CP""$CP_SEPARATOR""liquibase_libs/"
+    for i in liquibase_libs/*.jar; do
+      CP="$CP""$CP_SEPARATOR""$i"
+    done
+  fi
 
   CP="$CP""$CP_SEPARATOR""$LIQUIBASE_HOME/lib/"
   for i in "$LIQUIBASE_HOME"/lib/*.jar; do
```

Comment:
We should probably do whatever if block we do for the liquibase_libs into here also? There should usually be a liquibase_home/lib dir, but I maybe remember seeing the same "can't load from classpath .../lib/*.jar" log message too? 

I don't have an example of what we were sometimes seeing off hand to know for sure, though.